### ISSUE TITLE
Add unit of measurement values on the Zorro telemetry screen

### ISSUE THREAD

**Issue by pfeerick** [EdgeTX/edgetx#3508]
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

When the telemetry from Multiplex is displayed on the RadioMaster Zorro 4-1 telemetry screen, there are no unit of measurement values after the numeric value.

![zorro](https://user-images.githubusercontent.com/131524236/233777084-88cd2035-270d-4d3a-89a7-23807943d883.jpg)

### Describe the solution you'd like

It would be nice to have the unit of measurement after the numeric valie, for example 7.5**V**, 11.0**V** and 19.1 **°C** (or°F) and other possible units such as **A** for current etc.

### Additional context

Originally incorrectly submitted as #3507

**Comment by 3djc** [comment]
Issue is screen resolution, not much miracles can be done on 128x64 pixels displays

**Comment by pfeerick** [comment]
Would first need to consider what the longest sensor name can be, and then how many digits the sensor values can go up to.

**Comment by mha1** [comment]
Can sensor names have more than 4 characters?

**Comment by pfeerick** [comment]
I just checked, and *at the present time*, no. However, the reason for that limit is no longer present, so that may change in the future.

Even so, when making UI changes, you need to consider the largest values, and work backwards...

![image](https://user-images.githubusercontent.com/5500713/236353341-60dc774d-01e5-4e1f-bbe5-c7c1d75ee08d.png)

While I think the Heading is a bit silly with two significant digits after the decimal, that is what it defaults to. 

Basically, I don't think units can just be added, with that particular screen layout. It would probably have to be a single column, not two columns, then there would be enough space to work with for all sensors without losing any precision.

**Comment by M-i-c-h-a-e-l1** [comment]
Ok, I get the point. I only have Multiplex telemetry sensors for temperature and battery voltage as this is for a trial truck. As shown on your picture above, this will probably not work if you have telemetry sensors which have more 4 or 5 digits. 

A work around could be, as you mention, to only have one column with 3 or 4 values. The Radiomaster Zorro can have up to 4 independent screens with telemetry values. This would however mean that one would manually press the 'Telem' button on the Zorro radio control to select the next telemetry screen. This is also done in other radios when there is not enough space on the screen for all telemetry values.
